brave-tests/src/main/java/brave/test/ITRemote.java

```diff
@@ -60,19 +60,22 @@
 
   /** Returns a trace context for use in propagation tests. */
   protected TraceContext newTraceContext(SamplingFlags flags) {
+    long id = System.nanoTime(); // Random enough as tests are run serially anyway
     TraceContext result = TraceContext.newBuilder()
-      .traceIdHigh(1L).traceId(2L).parentId(3L).spanId(1L)
+      .traceIdHigh(id).traceId(id + 1).parentId(id + 2).spanId(id + 3)
```

Comment:
I made a bug somewhere where I called this twice and expected different results :)